ui/js/directives/treeherder/clonejobs.js

```diff
@@ -274,10 +274,9 @@ treeherder.directive('thCloneJobs', [
                     ev.preventDefault();
                     //Middle mouse button pressed
                     ThJobModel.get(this.repoName, job.id).then(function(data){
-                        //Retrieve the job reference data and open a new
-                        //window on the job's log file
-                        if(data.logs.length > 0){
-                            window.open(data.logs[0].url, "Log");
+                        // Open the logviewer in a new window
+                        if (data.logs.length > 0 && data.state === 'completed') {
```

Comment:
I don't think incomplete jobs can have a log URL.
The log viewer button elsewhere in the UI does a check on `job_log_url.parse_status == 'parsed'` but I don't think we have that available to us here. 
Let's maybe drop the check for now?